opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -343,7 +343,7 @@ void ParserState::handleRandomText(const string_view& keywordString ) const {
             << this->current_path()
             << ":" << this->line();
     }
-
+    deck->getMessageContainer().warning(msg.str());
```

Comment:
``msg`` is ``std::stringstream``, it have already contain the filename and line number I guess?